### DISCUSSION TITLE
智慧車聯網🚘

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS3073-C/Python-Community-GS3073-C#135] [General]
**新聞內容:**
物聯智慧 (6565-TE) 與廣達電腦合作開發「智慧車聯網解決方案」。有助車載資通訊系統服務運營商，推出具備車規等級的影像，開創車用物聯網市場商機。
採用廣達電腦研發的 5G AI 多鏡頭行車記錄器，其內建邊緣運算、5G 聯網及 AI 影像擷取及辨識功能，將攝影畫面嵌入至物聯智慧的雲端影像管理系統 (Cloud VMS)。
搭配物聯智慧的網路即時通訊技術 (WebRTC)，讓管理者透過行動裝置即可查看旗下車輛的即時監控畫面或回放，無時差掌握狀況，並可透過 AI 影像辨識，自動偵測駕駛人狀態、道路路況，並具備自動觸發事件提醒與錄影功能，將重要影像留存雲端。
此方案可應用於物流業、車輛租賃服務業、計程車隊、城市公共交通運輸系統等領域，並讓車載資通訊系統整合服務或運營商，有效達成各種管理需求，此外，雲端的事件錄影畫面回放影像，大大節省事件處理的人力與時間成本。

**新聞心得:**
我覺得即時監控並使用AI辨識，偵測事件當下駕駛人以及路面狀況，對於交通產業的業者都能更方便的運用科技來掌握情況，而且還有防盜碰撞事件推播、疲勞駕駛示警這兩項功能，不僅能保障駕駛人在路上行駛的狀況，也對於車體維護多加一份安心。

**新聞來源:**
https://news.cnyes.com/news/id/4777610?exp=a
![4cfe5f73ffcdd4f88a4ef369f1a680db](https://user-images.githubusercontent.com/90883526/144611967-2e076350-0e8e-4cba-bf5e-8d81de840ddd.jpg)

**Comment by Timmy110403540** [comment]
車聯網、自動駕駛等高科技的技術，將會大幅減少車禍等意外發生。但當然駕駛者也要時時保持警覺，不可全依賴科技。

**Comment by briankuo123** [comment]
這種聯網的行車紀錄器應該會因為有偵測駕駛人狀態而產生隱私問題，不知道他們打算怎們解決

**Comment by pocper1** [comment]
有些駕駛買了自駕車，就會在駕駛座上面睡覺(還真的有人敢在開車時睡覺)，如果能偵測駕駛人的狀況，我想也還不錯~

**Comment by wh20502** [comment]
最近有一些事故，像是有自駕車偵測到前方明明突然出現物體，但是為了怕緊急煞車害駕駛受傷而選擇撞過去，雖然人為操作一樣會撞到，但是責任歸屬就會比較麻煩。

**Comment by jack90325** [comment]
雖然自動偵測非常的方便，但是如果出現緊急狀況還是需要手動駕駛

**Comment by yasklop** [comment]
最近自駕的發展真的非常的迅速，以後是不是可以不用考駕照了

**Comment by ghost** [comment]
感覺自駕是一個很危險的東西 即使未來大大推廣流行 我應該也不敢XD

**Comment by Shiba010** [comment]
感覺在安全層面上是個問題，如果以後這個系統遭受駭客攻擊，會不會路上使用自駕的用戶死傷慘重....

**Comment by 110409538** [comment]
我覺得科技足夠成熟後，將可以帶來大量的好處。
